Frontend/app/src/main/java/io/happybugs/happybugs/ReportActivity.java

```diff
@@ -49,76 +54,60 @@ protected void onCreate(Bundle savedInstanceState) {
-                        editTexts.whenText, editTexts.whoText, editTexts.detailsText);
-                enableBoxes(editTexts.whereText, checkBoxes.whereCheck, buttons.saveBtn);
-                enableView(editTexts.whereText, views.whereView,
-                        views.whatView, views.whenView, views.whoView, views.detailsView);
+                questionClickEvent(editTexts, editTexts.whereText, checkBoxes.whereCheck,
+                        buttons.saveBtn, views, views.whereView);
                 break;
             case R.id.whenBtn:
-                enableTextView(editTexts.whenText, editTexts.whatText,
-                        editTexts.whereText, editTexts.whoText, editTexts.detailsText);
-                enableBoxes(editTexts.whenText, checkBoxes.whenCheck, buttons.saveBtn);
-                enableView(editTexts.whenText, views.whenView,
-                        views.whatView, views.whereView, views.whoView, views.detailsView);
+                questionClickEvent(editTexts, editTexts.whenText, checkBoxes.whenCheck,
+                        buttons.saveBtn, views, views.whenView);
                 break;
             case R.id.whoBtn:
-                enableTextView(editTexts.whoText, editTexts.whatText,
-                        editTexts.whereText, editTexts.whenText, editTexts.detailsText);
-                enableBoxes(editTexts.whoText, checkBoxes.whoCheck,buttons.saveBtn);
-                enableView(editTexts.whoText, views.whoView,
-                        views.whatView, views.whereView, views.whenView, views.detailsView);
+                questionClickEvent(editTexts, editTexts.whoText, checkBoxes.whoCheck,
+                        buttons.saveBtn, views, views.whoView);
                 break;
             case R.id.detailsBtn:
-                enableTextView(editTexts.detailsText, editTexts.whatText,
-                        editTexts.whereText, editTexts.whenText, editTexts.whoText);
-                enableBoxes(editTexts.detailsText, checkBoxes.detailsCheck, buttons.saveBtn);
-                enableView(editTexts.detailsText, views.detailsView,
-                        views.whatView, views.whereView, views.whenView, views.whoView);
+                questionClickEvent(editTexts, editTexts.detailsText, checkBoxes.detailsCheck,
+                        buttons.saveBtn, views, views.detailsView);
                 break;
         }
     }
 
-    //Open and collapse edit text on button click
-    protected void enableTextView(TextView tv1, TextView tv2,
-                                  TextView tv3, TextView tv4, TextView tv5){
-        if (tv1.getVisibility() == View.GONE) {
-            tv2.setVisibility(View.GONE);
-            tv3.setVisibility(View.GONE);
-            tv4.setVisibility(View.GONE);
-            tv5.setVisibility(View.GONE);
-            tv1.setVisibility(View.VISIBLE);
+    public void questionClickEvent(ReportEditTexts editTexts, EditText editText,
```

Comment:
Can you add a comment about what this method is doing? It's not so trivial to infer from the method name itself.